OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/CanonicalizedSecretKey.java

```diff
@@ -117,6 +117,26 @@ public boolean isUsable() {
             return this != UNAVAILABLE && this != GNU_DUMMY;
         }
 
+        /** Compares by "usability", which basically compares how independently usable
+         * two SecretKeyTypes are.
+         */
+        public int compareUsability(SecretKeyType other) {
```

Comment:
on second thought, I chose the term usability because this is explicitly *not* capability. it compares two keys which have the same capabilities (sign/encrypt/certify) and orders them by how indepently usable they are, i.e.: "empty passphrase < passphrase/others < divert < stripped"

maybe the word "usability" isn't perfect because it is used in a different sense than UX, but "capability" is defined more sharply in the context of keys, so I want to avoid that word.